### PR TITLE
DocAI connector

### PR DESCRIPTION
**Pull request by ben741**

### CODE REVIEW

File: certified-connectors/Zuva DocAI/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+- `Create Document Classification Request`: Initiate asynchronous classification of the document type.
+- `Get Document Classification Request Status`: Get status and results of document classification.
+
+### Optical Character Recognition (OCR)
+
+Obtain the text of your documents, regardless of their original file type (pdf, docx, png ...), as well
+as images of the document.
+
+- `Create OCR Request`: Initiate synchronous OCR processing of a file.
```
Comment (matt-cote) [inline certified-connectors/Zuva DocAI/README.md]: Shouldn't this be `asynchronous`?
```suggestion
- `Create OCR Request`: Initiate asynchronous OCR processing of a file.
```

File: certified-connectors/Zuva DocAI/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- `Get OCR Request Status`: Check whether OCR processing of a file is complete.
+- `Get OCR Results Text`: Retrieve the text from a processed file.
+- `Get OCR Results Images`: Retrieve the images from a processed file.
+
+### Fields
+
+Fiend out what fields available to you.
```
Comment (matt-cote) [inline certified-connectors/Zuva DocAI/README.md]:
```suggestion
Find out what fields are available to you.
```

File: certified-connectors/Zuva DocAI/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,837 @@
...
+    "version": "1.0.0",
+    "title": "Zuva DocAI",
+    "description": "Embed Contracts AI into your product workflows using Zuva DocAI. No need for AI development from the ground up, leverage the DocAI APIs to extract information from your contracts.",
+    "contact": {
+      "name": "Zuva Support",
+      "url": "https://zuva.ai/support/",
+      "email": "support@zuva.ai"
+    }
+  },
+  "host": "us.app.zuva.ai",
```
Comment (matt-cote) [inline certified-connectors/Zuva DocAI/apiDefinition.swagger.json:13]: You probably already looked into this, but just want to confirm: is it not possible to pass the host as a parameter, similar to the token?

If that is the case, I assume if we wanted to add the other regions in the future we would have to create separate connectors? Perhaps we consider updating the title to include the region, something like `"Zuva DocAI (US)"`?

Comment (ben741) [reply certified-connectors/Zuva DocAI/apiDefinition.swagger.json:13]: I did look into it and couldn't find a good way, but your rephrasing of the question led me to try again and I found the way to do it properly: https://docs.microsoft.com/en-us/connectors/custom-connectors/policy-templates/dynamichosturl/dynamichosturl

Comment (matt-cote) [reply certified-connectors/Zuva DocAI/apiDefinition.swagger.json:13]: That's great, nice find!

**Review by matt-cote — APPROVED** [review]

**Pull request merged**